src/controllers/update/update.director.nationality.controller.ts

```diff
@@ -5,7 +5,7 @@ import { getDirectorNationality, postDirectorNationality } from "../../controlle
 
 
 export const get = (req: Request, resp: Response, next: NextFunction) => {
-  getDirectorNationality(req, resp, next, Templates.UPDATE_DIRECTOR_NATIONALITY, UPDATE_DIRECTOR_NATIONALITY_PATH, true)
+  getDirectorNationality(req, resp, next, Templates.UPDATE_DIRECTOR_NATIONALITY, UPDATE_DIRECTOR_DETAILS_PATH, true)
```

Comment:
Might be worth adding a unit test for the back link

Reply:
Added xtra tests